packages/ses/scripts/bundle.js

```diff
@@ -4,6 +4,7 @@ import fs from 'fs';
 import { makeBundle } from '@endo/compartment-mapper/bundle.js';
 import { minify } from 'terser';
 import { fileURLToPath, pathToFileURL } from 'url';
+import packageJson from '../package.json' assert { type: 'json' };
```

Comment:
TODO: rewrite to fix `test-async-hooks (16.x, ubuntu-latest)` failing tests

```console
import packageJson from '../package.json' assert { type: 'json' };
                                          ^^^^^^

SyntaxError: Unexpected identifier
    at Loader.moduleStrategy (node:internal/modules/esm/translators:146:18)
    at async link (node:internal/modules/esm/module_job:67:21)
```

Reply:
fixed: https://github.com/endojs/endo/pull/1854/commits/c14ff1c2ea1490558cb8073a0158708558d683ae